src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/crud-table/crud-table.component.html

```diff
@@ -62,3 +62,14 @@
              let-value="value">
   <span>{{ value | duration }}</span>
 </ng-template>
+
+<ng-template #authxExportModal>
+  <div class="d-flex flex-column align-items-center w-100 gap-3">
+    <textarea readonly
+              class="w-100 bg-light height-400"
+              id="authExportArea">{{ modalState.authExportData }}</textarea>
+    <cd-copy-2-clipboard-button class="align-self-end"
+                                source="authExportArea">
+    </cd-copy-2-clipboard-button>
```

Comment:
an option to download the data is also good.

We have the `cd-download-button` component.

if we have normal data then,
```
    <cd-download-button [textItem]="data"
                        fileName="authExportArea">
```

if we have json data then,

```
    <cd-download-button [objectItem]="jsonData"
                        fileName="authExportArea">
```

Can we use this as well here?